nbviewer/handlers.py

```diff
@@ -103,7 +105,14 @@ def init_handlers(formats, providers, base_url, localfiles):
     handlers = provider_handlers(providers)
 
     # Add localfile handlers if the option is set
-    handlers = [(r'/localfile/?(.*)', LocalFileHandler)]+handlers if localfiles else handlers
+    if localfiles:
+        pre_providers = pre_providers + [
+            # support for local *.html files (in iframes) passes through StaticFileHandler
+            (r'/localfile/?(.*\.html)', web.StaticFileHandler, {'path': os.path.abspath(localfiles)}),
```

Comment:
I haven't dug in, but it's possible this rule is matching against a path that was originally handled by the slide show handler but is now handled by the StaticFileHandler.